Client/core/CGraphStats.cpp

```diff
@@ -199,13 +202,18 @@ void CGraphStats::Draw()
         return;
 
     CGraphicsInterface* pGraphics = g_pCore->GetGraphics();
+    CLocalGUI*          pLocalGUI = g_pCore->GetLocalGUI();
+
+    uint  uiViewportWidth = pGraphics->GetViewportWidth();      // get width of current resolution
+    uint  uiViewportHeight = pGraphics->GetViewportHeight();    // get height of current resolution
+    uint  uiOriginX = 10;                                       // offset the graph by 10 pixels from left side of screen
+    uint  uiOriginY = pLocalGUI->GetChatBottomPosition();       // get chat bottom screen position
+    uint  uiSizeX = uiViewportWidth / 4;                        // set the width of graph to 1/4 of current resolution
+    uint  uiSizeY = uiViewportHeight / 4;                       // set the height of graph to 1/4 of current resolution
+    uint  uiRangeY = 100;                                       // 100ms
```

Comment:
`std::uint32_t`

Reply:
Will any of these values ​​be greater than 65535?

Reply:
unlikely but you could force a resolution greater than 65535. also not sure if chat position setting itself got some bound checks.